python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1721.html

```diff
@@ -1,28 +1,20 @@
-<p>
-Parentheses are not required after the 
-<code>assert</code>, <code>del</code>, <code>elif</code>, <code>except</code>, <code>for</code>, <code>if</code>, 
-<code>in</code>, <code>not</code>, <code>raise</code>, <code>return</code>, <code>while</code>, and <code>yield</code> 
-keywords, and using them unnecessarily impairs readability. 
-They should therefore be omitted.</p>
-
+<p>Parentheses are not required after the <code>assert</code>, <code>del</code>, <code>elif</code>, <code>except</code>, <code>for</code>,
+<code>if</code>, <code>in</code>, <code>not</code>, <code>raise</code>, <code>return</code>, <code>while</code>, and <code>yield</code> keywords, and
+using them unnecessarily impairs readability. They should therefore be omitted.</p>
 <h2>Noncompliant Code Example</h2>
-
 <pre>
 x = 1
-while (x < 10):
+while (x &lt; 10):
     print "x is now %d" % (x)
     x += 1
 </pre>
-
 <h2>Compliant Solution</h2>
-
 <pre>
 x = 1
-while x < 10:
+while x &lt; 10:
     print "x is now %d" % (x)
     x += 1
 </pre>
+<h2>Deprecated</h2>
+<p>This rule is deprecated; use {rule:python:S1110} instead.</p>
```

Comment:
@ivandalbosco I think this rule should not be deprecated anymore: I added a comment on https://jira.sonarsource.com/browse/RSPEC-1110